splunk_logger.go

```diff
@@ -36,6 +37,7 @@ import (
 	"github.com/docker/docker/daemon/logger"
 	"github.com/docker/docker/daemon/logger/loggerutils"
 	"github.com/docker/docker/pkg/urlutil"
+	"github.com/fuyufjh/splunk-hec-go"
```

Comment:
i don't think we should be using this 4rd party splunk HEC client library when we have one inside the connector.

Reply:
can we use hec_client.go

Reply:
Now using hec_client.go